arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsManagementClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/alertrules/{ruleName}/incidents/{incidentName}": {
+      "get": {
+        "description": "Gets an incident associated to an alert rule",
+        "tags": [
+          "Alert"
```

Comment:
tags should match the Noun part of the operationId: Noun_Verb.
/cc @johanste @tbombach - If we decide to process tags in the future then do you guys think we would want tags to be consistent with Noun part of the operationId?